docs/output.md

```diff
@@ -268,6 +296,10 @@ See the [`--input`](usage.md#--input) section in the usage documentation for fur
   - CSV containing an entry for each sample with the columns `patient,sample,vcf`
   </details>
 
+#### Sentieon QualCal (BQSR)
+
+Currently, Sentieon's version of BQSR, QualCal, is not available in Sarek. Recent Illumina sequencers tend to provide well-calibrated BQs, so BQSR may not provide much benefit. By default Sarek runs GATK's BQSR; that can be skipped by adding the option `--skip_tools baserecalibrator`.
```

Comment:
I would not put this in the output docs to just tell someone, well it doesn't exist :D. Maybe rather to the FAQ about sentieon?

Reply:
Good point. I'll add a Sentieon paragraph at the bottom of the FAQ section in `usage.md`.